src/SqlPersistence.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -112,6 +112,7 @@ namespace NServiceBus
         public static void ConnectionBuilder(this NServiceBus.PersistenceExtensions<NServiceBus.Persistence.Sql.SqlPersistence> configuration, System.Func<System.Data.Common.DbConnection> connectionBuilder) { }
         public static void DisableInstaller(this NServiceBus.PersistenceExtensions<NServiceBus.Persistence.Sql.SqlPersistence> configuration) { }
         public static NServiceBus.Persistence.Sql.SagaSettings SagaSettings(this NServiceBus.PersistenceExtensions<NServiceBus.Persistence.Sql.SqlPersistence> configuration) { }
+        public static void Schema(this NServiceBus.PersistenceExtensions<NServiceBus.Persistence.Sql.SqlPersistence> configuration, string schema) { }
```

Comment:
AFAIK Schema in Oracle is more-or-less equivalent of Catalog in SQL Server: https://docs.microsoft.com/en-us/sql/ssma/oracle/mapping-oracle-schemas-to-sql-server-schemas-oracletosql